internal/web/app.go

```diff
@@ -39,24 +44,25 @@ func getFileSystem(useOS bool) http.FileSystem {
 	return http.FS(fsys)
 }
 
-type JobData struct {
-	Variant                string `json:"variant"`
-	Model                  string `json:"model"`
-	TrustedBoot            bool   `json:"trusted_boot"`
-	KubernetesDistribution string `json:"kubernetes_distribution"`
-	KubernetesVersion      string `json:"kubernetes_version"`
-	Image                  string `json:"image"`
-	Version                string `json:"version"`
-}
-
-func App(listenAddr, outDir string) error {
+func App(listenAddr, outDir, buildsDirectory string, config ...AppConfig) error {
```

Comment:
Would it make sense to put everything in the AppConfig instead of different parameters?

Reply:
Although I hate "config" arguments, in this one I think it makes sense. As long as it's only used for App initialization.

Reply:
done